oauth2_provider/migrations/0007_auto_20181019_0349.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 2.1.1 on 2018-10-19 06:49
```

Comment:
can you remove this file and pull from upstream and regen? or could you check the migrations failures? If you don't have time and have any fix in mind let me know, will push on your branch.